addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3156,6 +3162,12 @@ var SnippetsMenu = Widget.extend({
             return snippetEditor.__isStarted;
         }
 
+        // In translate mode, only allow creating the editor if the target is a
+        // text option snippet.
+        if (this.options.enableTranslation && !this._allowInTranslationMode($snippet[0])) {
+            return new Promise(resolve => resolve(null));
```

Comment:
I think you can do this:
```suggestion
            return Promise.resolve(null);
```